### ISSUE TITLE
Webview not accessible

### ISSUE THREAD

**Issue by EdmarS** [jnschulze/flutter-webview-windows#287]
The content of the webview is not accessible using a Screen Reader. Adding the Semantic widget does not improve the accessibility of the webview widget.
How can I make sure the webview widget is accessible?
